File: Programming Basics with Java/01. First Steps In Coding - Exercise/src/CharityCampaign.java
import java.util.Scanner;

public class CharityCampaign {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int days = Integer.parseInt(scan.nextLine());
        int cooks = Integer.parseInt(scan.nextLine());
        int cakes = Integer.parseInt(scan.nextLine());
        int waffles = Integer.parseInt(scan.nextLine());
        int pancakes = Integer.parseInt(scan.nextLine());

        double revenuePerDay = (cakes * 45 + waffles * 5.8 + pancakes * 3.2) * cooks;
        double revenuePerWhole = revenuePerDay * days;
        double profit = revenuePerWhole - revenuePerWhole / 8;

        System.out.printf("%.2f", profit);

    }
}

File: Programming Basics with J/08. More Exersices/charitycampaign.js
function charity(days, cooks, cakes, waffles, pancakes){
    days = Number(days);
    cooks = Number(cooks);
    cakes = Number(cakes);
    waffles = Number(waffles);
    pancakes = Number(pancakes);

    let cakeProfit = cakes * cooks * days * 45;
    let waffleProfit = waffles * cooks * days * 5.8;
    let pancakeProfit = pancakes * cooks * days * 3.2;

    let result = cakeProfit + waffleProfit + pancakeProfit;
    result -= result/8
    console.log(result);
}

charity("23", "8", "14", "30", "16");